src/components/images/imageLoader.js

```diff
@@ -132,15 +114,21 @@ import 'css!./style';
         }
         elem.setAttribute('data-src', url);
 
-        if (elem.classList.contains('non-blurhashable') || !userSettings.enableBlurhash()) {
-            elem.classList.remove('lazy-image-fadein-fast', 'lazy-image-fadein');
-            elem.classList.add('lazy-hidden');
-        } else {
-            switchCanvas(elem);
-        }
+        elem.classList.remove('lazy-image-fadein-fast', 'lazy-image-fadein');
+        elem.classList.add('lazy-hidden');
     }
 
     export function lazyChildren(elem) {
+        for (const lazyElem of elem.getElementsByClassName('lazy')) {
+            if (userSettings.enableBlurhash()) {
```

Comment:
@MrTimscampi Why we don't do this check outside the ``for`` loop?

Reply:
Because I copypasted code around :smile: